run.py

```diff
@@ -1,6 +1,8 @@
 # run.py
 
-from app import app
+from app import app_factory
+
+app = app_factory('DevelopmentConfig')
```

Comment:
You should not hard code environments, this should be picked from the shell environment. This makes your app more flexible.